src/components/search/SearchResultsPaging.jsx

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import Config from 'Config'
+import 'react-bootstrap-table-next/dist/react-bootstrap-table2.min.css'
+import Pagination from 'react-bootstrap/lib/Pagination'
+import { showSearchResults } from 'actions/index'
+
+const SearchResultsPaging = (props) => {
+  const [currentPage, setCurrentPage] = useState(1) // initialize currentPage to 1
+
+  // If there are no results, we don't need paging
+  if (props.totalResults === 0) {
+    return null
+  }
+
+  // If there are fewer results than one full page, we don't need paging
+  if (props.totalResults <= Config.searchResultsPerPage) {
+    return null
+  }
+
+  /* Event.target.text is the page number clicked on
+   * or:
+   *   « - first
+   *    ‹ - previous
+   *    › - next
+   *   » - last
+   */
+  const handleClick = (event) => {
+    let queryFrom = 0
+    let newCurrentPage = 1
+    switch (event.target.text) {
+      case '«':
+        break
+      case '‹':
+        newCurrentPage = currentPage - 1
+        break
+      case '›':
+        newCurrentPage = currentPage + 1
+        break
+      case '»':
+        newCurrentPage = props.totalResults / Config.searchResultsPerPage
+        break
+      case undefined: // this is required to capture clicks on disabled buttons
+        return
+      default:
+        newCurrentPage = Number(event.target.text)
+        break
+    }
+    queryFrom = (newCurrentPage - 1) * Config.searchResultsPerPage
+    setCurrentPage(newCurrentPage)
+    search(props.queryString, queryFrom)
+  }
+
+  const responseToSearchResults = json => ({
+    totalHits: json.hits.total,
+    results: json.hits.hits.map(row => ({
+      uri: row._id,
+      title: row._source.title,
+    })),
+  })
+
+
+  const search = (query, queryFrom) => {
+    const uri = `${Config.searchHost}${Config.searchPath}?q=title:${query}%20OR%20subtitle:${query}&from=${queryFrom}&size=${Config.searchResultsPerPage}`
```

Comment:
It seems odd that we have this knowledge in here and in `components/search/Search`. Can this functionality be extracted?